src/main/java/net/atos/zac/policy/output/TaakRechten.java

```diff
@@ -22,11 +23,13 @@ public TaakRechten(
             @JsonbProperty("lezen") final boolean lezen,
             @JsonbProperty("wijzigen") final boolean wijzigen,
             @JsonbProperty("toekennen") final boolean toekennen,
+            @JsonbProperty("creeeren_document") boolean creeerenDocument,
```

Comment:
Per other comment of underscore word separation in Json properties !

Reply:
Closing this in favour of the previous convo/discussion on the topic